test/integration/multiplexed_upstream_integration_test.cc

```diff
@@ -573,6 +573,88 @@ TEST_P(MultiplexedUpstreamIntegrationTest, LargeResponseHeadersRejected) {
   EXPECT_EQ("503", response->headers().getStatusValue());
 }
 
+// Tests configuration of max response headers size.
+TEST_P(MultiplexedUpstreamIntegrationTest, LargeResponseHeadersAccepted) {
```

Comment:
I started with adding this test, and while debugging it realized that the Multiplexed tests don't have any http1 variants, so I moved over to protocol_integration_test. I found related tests in both files, and I don't understand why both are needed? Do you know the purpose of the `MultiplexedUpstreamIntegrationTest` tests?

Reply:
protocol-only is fine IMO

Reply:
removed multiplexed-upstream tests